gef-ui/server/controllers/application-details/index.js

```diff
@@ -133,6 +133,8 @@ function buildBody(app, previewMode, user) {
     canUpdateUnissuedFacilities: canUpdateUnissuedFacilitiesCheck(app, unissuedFacilitiesPresent, facilitiesChangedToIssued, hasUkefDecisionAccepted),
     MIAReturnToMaker: isMIAWithoutChangedToIssuedFacilities(app),
     returnToMakerNoFacilitiesChanged: returnToMakerNoFacilitiesChanged(app, hasChangedFacilities),
+    dealStatusCancelled: app.status === DEAL_STATUS.CANCELLED,
```

Comment:
```suggestion
    const dealCancelledStatus = [DEAL_STATUS.CANCELLED, DEAL_STATUS.PENDING_CANCELLATION];
    ...
    canCloneDeal: !dealCancelledStatus.includes(app.status),
```

Above will make code readable, concice and abstractable.